docs/user/reference/sitemaps.rst

```diff
@@ -31,4 +31,11 @@ Custom ``sitemap.xml``
 You can control the sitemap that is used via the ``robots.txt`` file.
 Our :doc:`/reference/robots` allows you to host a custom version of this file.
 
+An example would look like::
+
+  User-agent: *
+  Allow: /
+
+  Sitemap: https://www.example.com/sitemap.xml
```

Comment:
Our users might be confused how to get the file at the root path like this. It could be more relatable by giving an example URL that includes version/language:

```suggestion
  Sitemap: https://www.example.com/en/stable/sitemap.xml
```